decidim/decidim
# Bugfix - moderated meetings are displayed in the meetings index page

#### :tophat: What? Why?
Adding the moderation parameter that hides the moderated meetings from directory controller.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #?
- Fixes #?

#### Testing
To repoduce:
- Go to meeting directory endpoint /meetings 
- Select a meeting and report it
- Go to admin and choose hide
- visit the meeting directory and the meeting is displayed. 
- Navigate to that particular meeting and you will get a 404 error 
To test fix: 
- Go to meeting directory endpoint /meetings 
- Select a meeting and report it
- Go to admin and choose hide
- visit the meeting directory and the meeting is no longer displayed. 

#### :clipboard: Checklist
:rotating_light: Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.


### :camera: Screenshots
*Please add screenshots of the changes you're proposing*
![Description](URL)

:hearts: Thank you!
